Locale::Gettext exports by default the various LC_* constants as does the
POSIX perl module. Up to perl-5.10, their definition was strictly the same
and didn't cause any harm. Now the POSIX module evolved slightly and the
symbol redefinition are conflictual and
generate warnings.

Resolve this by making sure that Locale::Gettext reexports the constants
coming from the POSIX module.

Fixes Debian bug #479803.

 -- Raphael Hertzog <hertzog@debian.org>

Index: liblocale-gettext-perl-1.05/gettext.pm
===================================================================
--- liblocale-gettext-perl-1.05.orig/gettext.pm	2008-05-07 09:40:23.000000000 +0200
+++ liblocale-gettext-perl-1.05/gettext.pm	2008-05-07 09:41:04.000000000 +0200
@@ -32,6 +32,7 @@
 =cut
 
 use Carp;
+use POSIX qw(:locale_h);
 
 require Exporter;
 require DynaLoader;
